Content.Server/Zombies/PendingZombieComponent.cs

```diff
@@ -14,16 +14,51 @@ public sealed class PendingZombieComponent : Component
     {
         DamageDict = new ()
         {
-            { "Blunt", 0.5 },
-            { "Cellular", 0.2 },
+            { "Blunt", 0.8 },
             { "Toxin", 0.2 },
         }
     };
 
     [DataField("nextTick", customTypeSerializer:typeof(TimeOffsetSerializer))]
     public TimeSpan NextTick;
 
-    // Scales damage over time.
-    [DataField("infectedSecs")]
+    /// <summary>
+    /// Scales damage over time.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("infectedSecs")]
     public int InfectedSecs;
+
+    /// <summary>
+    /// Number of seconds that a typical infection will last before the player is totally overwhelmed with damage and
+    ///   dies.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField("infectionLength")]
+    public float InfectionLength = 120f;
```

Comment:
should prob make this MaxInfectionLength and make it 4 mins and to a Random(0, comp.MaxInfectionLength) once we are infected

Reply:
Will do. Btw if it is set to 0 there will be a div/0

Reply:
oh right, you do a division with it later, yeah, set the minimum to 1 (tho the likelihood to roll a flat 0 is absurdly low i would imagine)